apps/nextjs-website/src/components/Feature.tsx

```diff
@@ -0,0 +1,29 @@
+'use client';
+import { Feature as FeatureEC } from '@pagopa/pagopa-editorial-components';
+import { FeatureProps } from '@pagopa/pagopa-editorial-components/dist/components/Feature/Feature';
+import { FeatureSection } from '@/lib/fetch/types/PageSection';
+
+const makeFeatureProps = ({
+  items,
+  background,
+  ...rest
+}: FeatureSection): FeatureProps => ({
+  items: items.map((item) => ({
+    icon: item.icon,
+    iconColor: item.iconColor,
+    title: item.title,
+    subtitle: item.subtitle,
+    linkText: item.linkText,
+    linkURL: item.linkURL,
```

Comment:
The attributes `icon`, `iconColor`, `linkText`, and `linkURL` should be defined in which type? I don't seem to have seen them in `FeatureProps`